### DISCUSSION TITLE
How can I make apps to listen on all network addresses?

### DISCUSSION THREAD

**Discussion by gerroon** [LykosAI/StabilityMatrix#948] [Q&A] [answered]
Hi

Thanks for this app. Is there a way to make the apps to listen in on all network devices, not just the local network so that I can access them on my network?

**Comment by mohnjiles** [comment, marked as answer]
It depends on the UI, most of them have a `--listen` option you can enable in SM's Launch Options menu (the ⚙️ icon on the package cards). A couple may not follow the standard, like InvokeAI, which is `--host` I believe.
